dgva-server/src/main/resources/application.properties

```diff
@@ -0,0 +1,19 @@
+# Configure JDBC datasource OR JNDI datasource
+# DGVAPRO Datasource configuration
+spring.datasource.url=@dgvapro.host@
+spring.datasource.username=@dgvapro.user@
+spring.datasource.password=@dgvapro.passwd@
+spring.datasource.driver-class-name=oracle.jdbc.OracleDriver
+spring.datasource.type=oracle.jdbc.pool.OracleDataSource
+spring.jpa.database-platform=org.hibernate.dialect.Oracle10gDialect
+
+#Avoid hibernate ddl schema generation/update or validation
+spring.jpa.hibernate.ddl-auto = none
+#spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.OracleDialect
```

Comment:
Could it be moved to the bottom for more clear grouping of properties (`spring.datasource` vs `spring.jpa`)? Also, please check if "OracleDialect" (without "10g") also works as it will be more generic and could still work in case of upgrade.